storage-node/packages/colossus/paths/asset/v0/{id}.js

```diff
@@ -21,15 +21,30 @@
 const debug = require('debug')('joystream:colossus:api:asset')
 const filter = require('@joystream/storage-node-backend/filter')
 const ipfsProxy = require('../../../lib/middleware/ipfs_proxy')
+const assert = require('assert')
 
 function errorHandler(response, err, code) {
   debug(err)
   response.status(err.code || code || 500).send({ message: err.toString() })
 }
 
+const PROCESS_UPLOAD_BALANCE = 3
```

Comment:
Do you mind documenting this constant? Also, why do we choose this exact value? FYI: we are implementing existential deposit in the Olympia - it could be higher than this value.

Reply:
This is the expected fees for all the 3 transactions that will need to be sent after processing the upload. Will document.
I don't think existential deposit is actually relevant. In-fact in Olympia we need to figure out how clients applications can compute fees ahead of time for transactions, but that is discussion for another time :)

Reply:
done in 8d08ccc78a1b6ad919f137d988f4fc9c29228baa